dbt/include/tidb/macros/materializations/snapshot/strategies.sql

```diff
@@ -6,26 +6,41 @@
     {%- endfor -%}))
 {%- endmacro %}
 
-{% macro snapshot_check_all_get_existing_columns(node, target_exists) -%}
-    {%- set query_columns = get_columns_in_query(node['compiled_sql']) -%}
+-- copy from dbt-core v1.2, just alter database=None in adapter.get_relation
```

Comment:
@shiyuhang0 what if a relation object never printed the database? would save you from having to override core's version of this macro?

Reply:
I think maybe we can delete this override because I found that the database has been set to None in get_relation. 
And I will test if it still works after delete the macro

Reply:
https://github.com/pingcap/dbt-tidb/pull/21